client/my-sites/email/paths.js

```diff
@@ -298,3 +298,31 @@ export function emailManagementInbox( siteName = null ) {
 export function isUnderEmailManagementAll( path ) {
 	return path?.startsWith( emailManagementAllSitesPrefix + '/' );
 }
+
+/**
+ *
+ * @param { string } ownerUserName The username that will appear in the log in screen
+ * @param { string | null | undefined } selectedSiteSlug The selected site slug
+ * @param { string } selectedDomainName The domain name to purchase email subscription
+ * @returns {string} Login Url with email_address and redirect_to parameters included in the URI
+ */
+export function loginUrlWithUserNameAndRedirectToEmailProvidersComparison(
+	ownerUserName,
+	selectedSiteSlug,
+	selectedDomainName
+) {
+	const userNameUrlParameter = `?email_address=${ ownerUserName }`;
+	const redirectUrlParameter = encodeURIComponent(
+		emailManagementPurchaseNewEmailAccount(
+			selectedSiteSlug ?? '',
+			selectedDomainName,
+			null,
+			'login-redirect',
+			'google-workspace'
+		)
+	);
+	const loginUrl = `https://wordpress.com/log-in${
```

Comment:
```suggestion
	const loginUrl = `/log-in${
```
Should we drop the origin here? Should make things easier during development.

Reply:
Right!